templates/registration/logged_out.html

```diff
@@ -0,0 +1,5 @@
+{% extends 'base.html' %}
+{% block content %}
+  <h2>You have been logged out</h2>
+  <p><a href="{% url 'login' %}">Log in again</a></p>
```

Comment:
Ensure that the URL name 'login' is defined in your URL configurations. If not, update the URL name to match the actual name used in your URL configurations.